README.md

```diff
@@ -20,6 +20,9 @@ This is the official Java implementation of the [Dynatrace OneAgent SDK](https:/
  	* [Trace incoming and outgoing remote calls](#remoting)
  	* [In process linking](#inprocess)
  	* [Add custom request attributes](#scav)
+ 	* [Trace web requests](#webrequests)
+ 	 	* [Trace incoming web requests](#inwebrequests)
+ 	 	* [Trace outgoing web requests](#outwebrequests)
```

Comment:
to keep it consistent with "trace incoming and outgoing remote calls", I suggest to use as title "trace incoming and outgoing web requests" or to introduce the same structure for remote calls.

Reply:
remotecalls (in and out) should only be used together. they have therefore only one section, where the whole usage will be explained.
in most cases i would say, webrequests will be used only on the outgoing or incoming side. and should splitted therefore.

Reply:
makes sense - ok for me to leave it that way